{{cookiecutter.project_name}}/pyproject.toml

```diff
@@ -7,7 +7,7 @@ authors = ["{{ cookiecutter.project_domain }}"]
 
 
 [tool.poetry.dependencies]
-python = "3.9.13"
+python = "3.9.*"
```

Comment:
Using unspecified patch version can lead to very strange bugs. Because python solve different bugs / security problems in patch releases. Some that works on `3.9.0` might not work the same way on `3.9.1`